packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js

```diff
@@ -173,6 +173,10 @@ export class ModelPage extends React.Component { // eslint-disable-line react/pr
   handleEditAttribute = (attributeName) => {
     const index = findIndex(this.props.modelPage.model.attributes, ['name', attributeName]);
     const attribute = this.props.modelPage.model.attributes[index];
+
+    if (attribute.params.type === 'enumeration') {
```

Comment:
It would be great to explain why there is an exception for the enumeration type.